app/views/DataView.js

```diff
@@ -58,11 +60,13 @@ module.exports = function (Marionette, Backbone) {
       this.last_received_date = new Date();
       this.render();
     },
-
+	
+	
     toggleKeepLastPacket: function(){
       this.keep_last_packet = !this.keep_last_packet;
     },
-
+	/**/
```

Comment:
remove these. NO changes were made in this file, so make sure none are commited, as you have a lot of space stuff here